tests/testthat/data/sample_config.json

```diff
@@ -48,8 +48,8 @@
         }
     },
     "sampler_opts": {
-        "cores": 4,
-        "chains": 4,
+        "cores": 1,
+        "chains": 1,
```

Comment:
Can you explain the implications?

Reply:
I don't think I can leave a comment in the file because JSON doesn't support comments, but the context here is that the GHA runners are single node VMs. A test with 4 cores works locally but not in Actions.